src/Features/CSharp/Portable/MakeStructFieldsWritable/CSharpMakeStructFieldsWritableDiagnosticAnalyzer.cs

```diff
@@ -0,0 +1,102 @@
+﻿
```

Comment:
copyright.

Reply:
Resolved in [https://github.com/dotnet/roslyn/pull/32063/commits/5f00e097f529b82deebe0783a03448f9c954f6ec](https://github.com/dotnet/roslyn/pull/32063/commits/5f00e097f529b82deebe0783a03448f9c954f6ec)